package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "contexture-react",
-  "version": "1.13.0",
+  "version": "1.13.1",
```

Comment:
this should be a minor release not fix revision

Reply:
ok